Doc/c-api/arg.rst

```diff
@@ -410,8 +410,8 @@ inside nested parentheses.  They are:
 
 ``$``
    :c:func:`PyArg_ParseTupleAndKeywords` only:
-   Indicates that the remaining arguments in the Python argument list are
-   keyword-only.  Currently, all keyword-only arguments must also be optional
```

Comment:
I think the old wording is clearer here, "keyword-only" is a well-known term in Python.